ssz-rs/src/merkleization/node.rs

```diff
@@ -1,9 +1,14 @@
 use crate::{lib::*, prelude::*, utils::write_bytes_to_lower_hex};
 
+use super::BYTES_PER_CHUNK;
+
+// TODO: Should we change Node's length to BYTES_PER_CHUNK?
+// They need to match for merkleization in ssz-rs-derive's hash_tree_root to work.
```

Comment:
yes, this was an oversight, perhaps I defined `Node` before I had `BYTES_PER_CHUNK` to rely on